lib/components/term.js

```diff
@@ -177,7 +177,20 @@ export default class Term extends Component {
   }
 
   clear() {
-    this.term.onVTKeystroke('\f');
+    // Command Running
+    if (this.term.getCursorColumn() === 0) {
```

Comment:
Seems really hacky.
I haven't test it but I think if your command is currently printing a progress bar, it could really break things.